### ISSUE TITLE
July Task: Checklist to Review GitHub Documentation Templates for Team PSD Manual 

### ISSUE THREAD

**Issue by ritahitching** [lzim/teampsd#1378]
- [ ] Create a checklist review GitHUb templates for **issues/Bugs** and **feature** cards.
- [ ] Link to the document [GitHub Documentation Checklist for Team PSD Manual](https://github.com/lzim/teampsd/blob/rita_2020_04_23_issue_1191/resources/training_guides/github/documentation_checklist_teampsd_manual.md) 
- [ ] Cross reference  #1192 Team PSD Manual - list of all documents for review
- [ ] Cross reference #1220 Develop Master Cross Walk - all variables
- [ ] Cross reference #1364 Bookdown style guide for headers - size for Bookdown manual and documents themselves

**Comment by lzim** [comment]
@ritahitching 

Thanks! It looks like you 👀  identified a number of important things in your list! 😅 
- I'd recommend that we look for ways to **simplify** and **streamline** 🔪  

- **See my comment in 2 below about a) getting the flow maps done correctly first, then b) cross -walking with the checklist, so a Team PSD member knows exactly what to do.**
- Once those are done correctly, please delegate a review process with instructions to divvy up the review via pull-requests to finalize.
- I believe @lijenn @staceypark @dlkibbe and @branscombj are all exceptional reviewers for clarity and ease, as well as the Team PSD graphics/style conventions. It just depends how much capacity folks have to wrap up these edits/review.

Below, I provide 5 leads that should keep the improvement iterations moving.
I can add more feedback as others do in this thread.

Rita, thanks for your work to move this forward!

---
1. The **link** labeled " Visual map on creating a new issue or bug" links to this "ZenHub Flow" https://github.com/lzim/teampsd/blob/master/resources/maps/maps_markdown/zenhub_flow.md, which a) _**does not depict a flow to help you decide whether something is an Issue or a Bug (see 2 below**),_ b) nor is it for ZenHub. 
- The Flow Map should begin with a Red Pill, followed by a decision diamond, like this one: https://github.com/lzim/teampsd/blob/master/resources/maps/maps_markdown/documentation_map.md

---
2. This **GitHub** _intermediate_ documentation would benefit from getting the **flow map** correct first (see 1 above).
- Once those are correct, the **Issue** and **Feature** card **checklists** should be edited to match.
- They should be simplified and aligned with one another. 

---
3.  Starting with an **action verb** "Check," or "Review", is often most clear and succinct, especially for a list of tasks.
- Here is an excerpt that sticks to this ~60% of the time (see **bold**). Follow that approach throughout.
- Consider which verb is most helpful, for example "**Open** [mtl.how/teampsd](https://mtl.how/teampsd) using your Internet browser." is clearer than "_Orient to..._"
### TeamPSD Difference Between Issue or Feature Cards
- **Orient** to MTL.how/teampsd on your internet browser
- In the panel select the Issues tab
- **Select** the New Issue green square on the right hand side
- **Decide** on the choice of ticket you want to create Feature or Issue
- Use case for Issue cards are when you have a task that you have been asked to do, or a Bug
---

4. We need to **standardize flow chart symbols.**
- I want to check whether the **Predefined Process** symbol for a "named process that is defined elsewhere" is more accurate than use of the **offpage connector**symbol, which implies the same process continues on another page https://www.zenflowchart.com/flowchart/
- Edit maps to remove needless visual complexity, e.g., remove "steps" in connectors that could simply be a straight line.
- NOTE: I also think there are also a number of other symbols (e.g., document, data, etc.) we should use to make our maps more easy to understand (e.g., the **RedCap** process needs these).
---

5. A note on **Lucid Charts** Templates:
- We should remove the VA Primary Dark Red #772432 from the Lucid Chart templates, as we did for all our Team PSD or _Modeling to Learn_ graphics. We have never used this color as it is very non-contemporatory. https://github.com/lzim/teampsd/blob/master/resources/maps/maps_markdown/zenhub_flow.md

These steps will really clean up these resources 🧹

**Comment by ritahitching** [comment]
- [ ] 1. Focus on finalizing the LucidChart maps.
- [ ] 2. Then cross-walking with the LucidChart maps with the finalised checklists for the current drafts of the:
- [Checklist for GitHub Documentation - Issue/Bug and Feature Cards](https://github.com/lzim/teampsd/blob/rita_2020_04_23_issue_1191/resources/training_guides/github/documentation_checklist_teampsd_manual.md) will need updating.
- [Define header level standards](https://github.com/lzim/teampsd/edit/rita_2020_04_27_issue_1364/resources/bookdown/header_level_standards_teampsd_documents.md) will need updating.- cross reference issue #1364

**Comment by lzim** [comment]
Yes, @ritahitching - This list of tasks looks good.
HOWEVER: To avoid rework, please cross-ref the notes I put on #1381, #1364, and #1192.
FYI: @staceypark @lijenn @branscombj @dlkibbe 

The key idea across the issues is the **ORDER OF OPERATIONS** 
- I would NOT continue doing these as unrelated discrete tasks.
- Rather, I would FINALIZE with Jane, Debbie and Jenn, the general principles that will guide all the other tasks.

In other words to get the **may_epic** cross-cutting priority to finish the **Team PSD Manual** #1192 and close all the open document cards in the **document_tracker**
1) finalize the **standardization rules** #1364 first, then once those are defined,
2) do the most overarching tasks that **further define Team PSD Project Management, or Team PSD Processes & Platforms,** like **GitHub** and our **Flow Maps** #1378 using those rules, and only then...
3) produce in a standard way the more **checklists and resources** that follow from 1) and 2) _meaning are style consistent with the overall manual, and follow the visual maps._

Does that make sense?

See the nice **Team PSD Manual** examples that I linked to on #1192 
*NICE Examples for Team PSD Manual:**
**Plain English for #1192**
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/orientation/repo-guidelines.md

With Maps
https://github.com/department-of-veterans-affairs/lighthouse-facilities

**Comment by ritahitching** [comment]
@lzim please confirm I can move this issue to the May Epic?
thanks

**Comment by lzim** [comment]
> @lzim please confirm I can move this issue to the May milestone & Epic?
> thanks

@ritahitching Yes, consistent with my “order of operations” instruction above, which we reviewed in the **hq_workgroup** meeting, #1364 is first priority. Then, once finalized, all the dependent issue cards can proceed according to the standardization rules.

**Comment by ritahitching** [comment]
@lijenn this is issue is blocked by the #1192 and the MVP process.
I recommend we keep this issue open.

**Comment by lijenn** [comment]
Closing as this is contains outdated work.
